src/components/learning-center/article-footer.tsx

```diff
@@ -65,31 +71,31 @@ const LearningArticleFooterScaffolding = (props: ContentfulContent) => {
   return (
     <div className="columns is-desktop has-background-white-ter">
       <div className="column footer-ctas">
-        <FooterCta content={props.content.learningCenterCta} />
-        <FooterCta content={props.content.justFixCta} />
+        <FooterCta locale={props.locale} content={props.content.learningCenterCta} />
+        <FooterCta locale={props.locale} content={props.content.justFixCta} />
       </div>
       <div className="column table-of-contents is-half-desktop">
         <div className="columns hero-body">
           <div className="column is-one-third">
             {ArticlesSortedByCategory.map( ( section: TableOfContentsSection, i: number ) =>
               (i % 3 === 0 ? 
-                <TableOfContentsSection key={i} noDivider={(ArticlesSortedByCategory.length <= i + 3)}
+                <TableOfContentsSection key={i} locale={props.locale} noDivider={(ArticlesSortedByCategory.length <= i + 3)}
```

Comment:
This might not be related to work done in this PR, but I'm curious where the number `3` is coming from here, and I wonder if it'd be helpful to have a named constant in its place?

Reply:
So all of the modulo arithmetic here is just to generate 3 columns of links in the table of contents section— hence the "3" here. Since this isn't really a property I designed to be dynamic, do you think it's ok to leave the 3 as is?

Reply:
Oh so I think doing a `const NUM_COLUMNS = 3` or `const NUM_COLUMNS_OF_LINKS = 3` at the top of the file could make this more readable, but it's up to you!  (My intent isn't to make the code more dynamic, just to make it easier to understand/modify if needed.)